### PR TITLE
!role list sorts output alphabetically

### PR DESCRIPTION
**Pull request by super-cooper**
This is a small change to allow !role list to sort its output alphabetically in both the argless and argumented use cases.

### CODE REVIEW

File: src/commands/role/list.py [hunk 1]
```diff
@@ -34,20 +34,15 @@ async def exec(self, args: typing.List[str], message: discord.Message) -> Comman
                     roles.append(role_obj.name)
                 elif memebot.client.user in role_obj.members:
                     can_manage = True
-            roles.insert(0, "Roles managed through `!role` command:")
-            return CommandOutput().set_text("\n- ".join(roles))
+            return CommandOutput(content=f"Roles managed through `!role` command:\n- " + "\n- ".join(sorted(roles)))
```
Comment (nhawke) [inline src/commands/role/list.py:37]: Another way to do this that might be cleaner and not crowd this one line is to sort before inserting the header item at the front of the list. Worth considering ot make this line not so busy, but up to you.

**Review by nhawke — CHANGES_REQUESTED** [review]

File: src/commands/role/list.py [hunk 2]
```diff
@@ -34,20 +34,15 @@ async def exec(self, args: typing.List[str], message: discord.Message) -> Comman
...
             if not target_role.members:
                 return CommandOutput().set_text(f'Role `@{target_name}` has no members!')
-            members_string = f'Members of `@{target_name}`:'
-            for member in target_role.members:
-                if member.nick is not None:
-                    members_string += f'\n- {member.nick} ({member.name})'
-                else:
-                    members_string += f'\n- {member.name}'
+            members_string = f'Members of `@{target_name}`:\n- ' + '\n- '.join(
+                sorted(member.nick if member.nick is not None else member.name for member in target_role.members))
```
[the remark this answers is not in the document]
Comment (super-cooper) [reply src/commands/role/list.py:46]: I didn't realize that I got rid of that functionality. Gonna fix it back.

**Review by super-cooper — COMMENTED** [review]
RFAL

**Review by nhawke — APPROVED** [review]

**Review by RobertWLeslie — APPROVED** [review]
lgt2m

**Pull request merged**